src/cards.less

```diff
@@ -0,0 +1,55 @@
+.card {
+  // Card border can't be changed with .border-x classes
+  // since we want to avoid the little blank space between
+  // image and border-top-left/right-radius, so we force
+  // the default border class and customize it on the img
+  .border;
```

Comment:
Maybe for cards, we don't need to use the crooked borders like shown in the other elements. It could be shown as something similar to a "Note Card" which has straight lines. That way the button within it stands out more. And then we don't have to worry about adjusting the image style from the border effects.

![image](https://user-images.githubusercontent.com/13633775/32299926-b74e6c80-bf2d-11e7-92a5-b3cae8692c9c.png)
![image](https://user-images.githubusercontent.com/13633775/32299994-fc69366a-bf2d-11e7-8991-356370f89b32.png) 

But I like the direction so far!

Reply:
Sure, I'll fix it asap. I have to admit that the crooked borders was not very good for the cards. I'll fix that and add a box shadow like on the pic, and also add header/footer customization on cards.